### DISCUSSION TITLE
How to include full MDX content in feed in gatsby-plugin-mdx v4

### DISCUSSION THREAD

**Discussion by danielpgross** [gatsbyjs/gatsby#37045] [Help] [answered]
From https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-plugin-mdx/README.md#v3-to-v4-breaking-changes:

>As mentioned above the html field was removed from the GraphQL node. We know that some of you used this for e.g. gatsby-plugin-feed. Unfortunately, for compatibility and performance reasons we had to remove it. We recommend using the excerpt field in the meantime until we find a feasible solution to provide MDX rendered as HTML. If you have any suggestions, please comment on the https://github.com/gatsbyjs/gatsby/discussions/25068.

How can we include full MDX content (not just an excerpt) in gatsby-plugin-feed items, now that the `html` field has been removed from MDX GraphQL nodes? Does a solution exist for this yet?

**Comment by timmb** [comment, marked as answer]
I've spent a day or so struggling with this, including trying to manually render the MDX on the needed pages in gatsby-node.js using `compileMDXWithCustomOptions` and manually creating a node field of the result. But the result is a string of javascript code and it's not clear how to make use of that without an `MDXRenderer` component.

I think I'll need to stick on Gatsby 4.20 until the the MDX plugin evolves.

**Comment by animanoir** [comment]
hey any solution for this yet? damn Gatsby is quite bad...

**Reply by mramirez0610** [reply]
ever find a solution?

**Reply by animanoir** [reply]
Yeah, I stopped using Gatsby!

**Reply by mramirez0610** [reply]
hahahaha well shit. portfolio is built on top of it, and i wanted to add a blog onto it, but rendering the body that's returned BACK into actual markdown is literally the biggest pain in the ass ever.

**Reply by animanoir** [reply]
Yeah my friend, I used to like Gatsby a lot, but literally seems like a dead project. I moved to Astro which I really recommend.
